packages/autorest.typescript/package.json

```diff
@@ -55,7 +55,7 @@
     "@autorest/codemodel": "~4.19.2",
     "@autorest/extension-base": "^3.5.0",
     "@autorest/testmodeler": "^2.4.0",
-    "@azure-rest/core-client": "^1.1.3",
+    "@azure-rest/core-client": "1.1.4-alpha.20230630.1",
```

Comment:
will change this back to `"@azure-rest/core-client": "^1.1.4"` once it's released.,